config/default/jenkins-infra.yaml

```diff
@@ -220,6 +220,23 @@ jenkins:
                     }
                   }
                 }
+            - script: >
+                multibranchPipelineJob('docker-rsyncd') {
```

Comment:
can't we just move to a github org folder? this is getting crazy updating this all the time?

Reply:
It's indeed probably a good time to switch

Reply:
here's an example of one https://github.com/hmcts/cnp-jenkins-config/blob/master/jobdsl/organisations-beta.groovy

Reply:
I've pushed https://github.com/jenkins-infra/charts/pull/733, as an attempt to configure the `jenkins-infra/docker-.*` repos in this way